sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1981,6 +1982,23 @@ class Dataset[T] private[sql](
   }
 
   /**
+   * Returns a new [[DataFrame]] that contains the result of applying a serialized R function
+   * `func` to each partition.
+   *
+   * @group func
+   */
```

Comment:
Spark 2.0 is a good change for add "since" for SparkR API methods. But I think we can do it consistently for all methods at one. I will submit a new JIRA issue for it.